experimental/single_number_formatter/src/provider.rs

```diff
@@ -0,0 +1,188 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// Provider structs must be stable
+#![allow(clippy::exhaustive_structs, clippy::exhaustive_enums)]
+
+//! Data provider struct definitions for this ICU4X component.
+//!
+//! Read more about data providers: [`icu_provider`]
+
+use alloc::borrow::Cow;
+use icu_provider::{yoke, zerofrom};
+use tinystr::TinyAsciiStr;
+use zerovec::{VarZeroVec, ZeroMap};
+
+#[icu_provider::data_struct(CurrencyEssentialV1Maker = "currency/essential@1")]
+#[derive(Default, Clone, PartialEq, Debug)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    databake(path = icu_singlenumberformatter::provider),
+)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+#[yoke(prove_covariance_manually)]
+pub struct CurrencyEssentialV1<'data> {
+    #[cfg_attr(feature = "serde", serde(borrow))]
+    pub indices_map: ZeroMap<'data, TinyAsciiStr<3>, CurrencyPatterns>,
+
+    // // #[cfg_attr(feature = "serde", serde(borrow))]
+    // pub currencies_patterns: VarZeroVec<'data, CurrencyPatternsULE>,
+    pub standard: Cow<'data, str>,
+    pub standard_alpha_next_to_number: Cow<'data, str>,
```

Comment:
Issue: These should both be `#[cfg_attr(feature = "serde", serde(borrow))]`

Reply:
done